server/src/main/java/org/elasticsearch/common/ReferenceDocs.java

```diff
@@ -202,7 +202,7 @@ static String getVersionComponent(String version, boolean isSnapshot) {
 
     @Override
     public String toString() {
-        return "https://www.elastic.co/guide/en/elasticsearch/reference/" + VERSION_COMPONENT + "/" + linksBySymbol.get(name());
+        return "https://www.elastic.co/docs/" + VERSION_COMPONENT + "/" + linksBySymbol.get(name());
```

Comment:
Since the URLs are handled differently for API docs and other docs, could we instead just pass in the info like this (and not use it in the API links)?

>        return linksBySymbol.get(name(VERSION_COMPONENT));

Or alternatively, not pass in the VERSION_COMPONENT at all and just have the full URL in the *.txt file, akin to what we're doing in the Elasticsearch specification repo in https://github.com/elastic/elasticsearch-specification/blob/main/specification/_doc_ids/table.csv?